### PR TITLE
Exclude global attributes settings to api-controller

### PR DESCRIPTION
**Pull request by donggyu04**
Make `GlobalControllerAdvice` work only on controller that return view (app.html)
> exclude api-controller, download controller

### CODE REVIEW

File: ngrinder-controller/src/main/java/org/ngrinder/home/controller/HealthCheckApiController.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import java.util.Map;
+
+/**
+ * Health check api controller.
+ *
+ * @since 3.5.0
+ */
+@Slf4j
+@RestController
+@GlobalControllerModel
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/home/controller/HealthCheckApiController.java]: Why is this necessary?

Comment (donggyu04) [reply ngrinder-controller/src/main/java/org/ngrinder/home/controller/HealthCheckApiController.java]: It my mistake 👍
